graviton/blackbox.py

```diff
@@ -938,7 +955,17 @@ def dig(self, dr_property: DryRunProperty, **kwargs: Dict[str, Any]) -> Any:
         ), f"{self.mode} cannot handle dig information from txn for assertion type {dr_property}"
 
         if dr_property == DryRunProperty.cost:
-            return txn["cost"]
+            # cost is treated as a derived property if budget-consumed and budget-added is available
```

Comment:
@tzaffi Optional:  Since we know `cost` is being removed, I don't see a reason to retain the secondary lookup (`txn["cost"]`).   Though I don't feel strongly.  If you prefer as is, feel welcomed to resolve.

Reply:
good catch